src/Tickets/Commerce/Gateways/Stripe/Webhooks.php

```diff
@@ -342,10 +495,78 @@ public function get_fields(): array {
 					'data-ajax-action-verify' => 'tec_tickets_commerce_gateway_stripe_verify_webhooks',
 				],
 			],
-			'tickets-commerce-gateway-settings-group-end-webhook'         => [
+			'tickets-commerce-gateway-settings-group-end-webhook' => [
 				'type' => 'html',
 				'html' => '<div class="clear"></div></div>',
 			],
 		];
 	}
+
+	/**
+	 * Get the description for the webhooks.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	protected function get_description_webhook_html(): string {
+		ob_start();
+		$kb_link = sprintf(
+			'<a target="_blank" rel="noopener noreferrer" href="%s">%s</a>',
+			esc_url( 'https://evnt.is/1b3p' ),
+			esc_html__( 'Learn more', 'event-tickets' )
+		);
+		?>
+		<p class="tec-tickets__admin-settings-tickets-commerce-gateway-group-description-stripe-webhooks contained">
+			<?php
+			printf(
+				// Translators: %s A link to the KB article.
+				esc_html__( 'Setting up webhooks will enable you to receive notifications on charge statuses and keep order information up to date for asynchronous payments. %s', 'event-tickets' ),
+				$kb_link // phpcs:ignore StellarWP.XSS.EscapeOutput.OutputNotEscaped, WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
1. **All** placeholders need to be indexed.
2. Move the $kb_link into this `printf` and you don't need to ignore phpcs

Reply:
Done!